data-prepper-plugins/aggregate-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/aggregate/AggregateActionInput.java

```diff
@@ -16,4 +18,11 @@ public interface AggregateActionInput {
      * @since 1.3
      */
     GroupState getGroupState();
+
+    /**
+     * @return a map of Identification keys containing identification keys and
+     *         their values
+     * @since 2.1
+     */
+    Map<Object, Object> getIdentificationKeys();
```

Comment:
Why did we rename this from `getIdentificationKeysMap` to `getIdentificationKeys`?